homeassistant/components/lutron_caseta/const.py

```diff
@@ -14,6 +14,9 @@
 
 BRIDGE_DEVICE_ID = "1"
 
+DEVICE_TYPE_WHITE_TUNE = "WhiteTune"
+DEVICE_TYPE_SPECTRUM_TUNE = "SpectrumTune"
```

Comment:
Are these available as constants from the pylutron-caseta lib?

Reply:
unfortunately not